### DISCUSSION TITLE
[Detector Support]: Any way to tell Coral it detected something wrong?

### DISCUSSION THREAD

**Discussion by ezcGman** [blakeblackshear/frigate#12426] [Detector Support] [answered]
### Describe the problem you are having

Hey there,

first of all: Thanks for Frigate, this is an awesome project! Finally an NVR that gives me reliable and accurate detections, 99.5% of the time.

About those 0.5%:
I do have two things that regularly / nearly daily get detected as a person:

- A satellite dish in my back yard, depending on how the light of a nearby traffic light reflects onto it:
![snapshot-1720829294 344117-o0m8ql~2](https://github.com/user-attachments/assets/cbd37172-0e85-453b-a16e-0c0ef5e5da8a)

- A lamp in my driveway, no idea when and why and how that spontaneously seems to be a person:
![snapshot-1720288511 083058-5wv8w3~2](https://github.com/user-attachments/assets/395c2d8c-e3d7-4444-b2ff-8271e5eb23d2)

Is there any way to help train the model / tell Coral it did something wrong to hopefully make it stop reporting these?

Thanks and greetings,

Andy!

### Version

0.13.2

### Frigate config file

```yaml
n/a
```

### docker-compose file or Docker CLI command

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Object Detector

Coral

**Comment by hawkeye217** [comment, marked as answer]
These are just [false positives](https://docs.frigate.video/frigate/glossary#false-positive) and are just because the free detection model Frigate uses (based on the COCO dataset) is not primarily trained on images from security cameras. See some examples [here](https://github.com/blakeblackshear/frigate/issues/8648#issuecomment-1817102926).

Frigate has several ways of dealing with false positives:
- Use [object filters](https://docs.frigate.video/configuration/object_filters/). If something is larger or smaller than it should be, you can use filters to suppress it. You could also use an object mask if a stationary object is always getting detected as a false positive.
- [Frigate+](https://docs.frigate.video/plus/). A premium service for Frigate that offers models trained on images submitted by Frigate+ users from their security cameras and is specifically designed for the way Frigate NVR analyzes video footage.

**Reply by ezcGman** [reply]
Aaah, I totally forgot about the object filters, thx. And this now also explains, what Frigate+ is for :)

I'll give both a look, but probably gonna try Frigate+ / giving something back :)

Thx!
